wolfssl/wolfcrypt/wc_port.h

```diff
@@ -365,7 +365,10 @@ WOLFSSL_API int wolfCrypt_Cleanup(void);
     WOLFSSL_API int wc_ReadDirNext(ReadDirCtx* ctx, const char* path, char** name);
     WOLFSSL_API void wc_ReadDirClose(ReadDirCtx* ctx);
 #endif /* !NO_WOLFSSL_DIR */
-
+#else
+    #ifdef WOLFSSL_ESPIDF
+        #include <stdio.h>
```

Comment:
Do we still need this include for `stdio.h`, now that you have logging functionality integrated?

Reply:
No, we don't need it anymore. I am going to remove this.